src/plots/gl2d/scene2d.js

```diff
@@ -359,7 +358,6 @@ proto.destroy = function() {
 
     if(!this.staticPlot) this.container.removeChild(this.canvas);
     this.container.removeChild(this.svgContainer);
-    this.container.removeChild(this.mouseContainer);
```

Comment:
Hmm. That probably breaks [`gl2d/camera.js`](https://github.com/plotly/plotly.js/blob/78b82b85206bde93c7117706278ac698b480139d/src/plots/gl2d/camera.js#L34)